src/components/graphs/TaskHoursByMonthGraph.tsx

```diff
@@ -217,7 +191,9 @@ function DataByMonthGraph() {
         tooltipConfig,
     ]);
 
-    return <div id="tasks-by-month" style={{ height: CARD_AREA_HEIGHT }} />;
+    return (
+        <div id="task-hours-by-month" style={{ height: CARD_AREA_HEIGHT }} />
```

Comment:
While we're here... let's throw this `id` into a `const` so we don't have to keep two strings in sync.

Reply:
I had the same thought, actually.